packages/labs/cli/src/lib/commands/generate.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {AbsolutePath} from '@lit-labs/analyzer/lib/paths.js';
+import {Command} from '../command.js';
+
+export const generate: Command = {
+  name: 'gen',
+  description: 'Generate Framework Wrappers',
```

Comment:
I think this can be sentence case - lowercase the other words

Reply:
Done